content/docs/user-guide/project-structure/pipelines-files.md

```diff
@@ -0,0 +1,416 @@
+individual [stages](/doc/command-reference/run) in one or more `dvc.yaml` files
+(or _pipelines files_). Stages form a pipeline when they connect with each other
+(forming a _dependency graph_, see `dvc dag`). Refer to
+[Data Pipelines](/doc/start/data-pipelines).
+
+> Note that a helper command, `dvc run`, is available to create (and execute)
+> stages.
+
+`dvc.yaml` files can be versioned with Git.
+
+These files use the [YAML 1.2](https://yaml.org/) file format, and a
+human-friendly schema explained below. We encourage you to get familiar with it
+so you may modify, write, or generate stages and pipelines on your own.
+
+> Note that we use [GNU/Linux](https://www.gnu.org/software/software.html) in
+> most of our examples.
+
+## Stages
+
+`stages` are named by the user and have a flexible structure. Here's a simple
+example:
+
+```yaml
+stages:
+  print2cols:
+    cmd: awk '{print $1 "\t" $2}' lines.txt > 2columns.txt
+    deps:
+      - lines.txt
+    outs:
+      - 2columns.txt
+```
+
+The most important part of a stage it's the terminal command(s) it executes
+(`cmd` field). This is what DVC runs when the stage is reproduced (see
+`dvc repro`).
+
+If a program executed by the command read input files, they can be defined as
+<abbr>dependencies</abbr> (`deps`). DVC will check whether they have changed to
+decide whether the stage requires re-execution (see `dvc status`).
+
+If the command writes files or directories, they can be defined as
+<abbr>outputs</abbr> (`outs`). DVC will track them going forward (similar to
+using `dvc add`).
+
+### Parameter dependencies
+
+[Parameters](/doc/command-reference/params) are a special type of stage
+dependency. They consist of a name/value pair to find in a YAML, JSON, TOML, or
+Python parameters file (`params.yaml` by default). Example:
+
+```yaml
+stages:
+  preprocess:
+    cmd: ./clean.sh raw.txt clean.txt
+    deps:
+      - raw.txt
+    params:
+      - threshold
+      - passes
+    outs:
+      - clean.txt
+```
+
+This allows several stages to depend on values of a shared structured file
+(which can be versioned directly with Git). See also `dvc params diff`.
+
+### Metrics and Plots outputs
+
+Like [common outputs](#outputs), <abbr>metrics</abbr> and <abbr>plots</abbr>
+files are produced by the stage `cmd`. However, their purpose is different.
+Typically they contain metadata to evaluate pipeline processes. Example:
+
+```yaml
+stages:
+  build:
+    cmd: python train.py
+    deps:
+      - features.csv
+    outs:
+      - model.pt
+    metrics:
+      - accuracy.txt:
+          cache: false
+    plots:
+      - auc.json:
+          cache: false
+```
+
+> `cache: false` is typical here, since they're small enough for Git to version
+> directly.
+
+The commands in `dvc metrics` and `dvc plots` help you display and compare
+metrics and plots.
+
+## Templating
+
+`dvc.yaml` supports a templating format to insert values from different sources
```

Comment:
for SEO purposes and so that users can find it-  probably makes sense to include parametrization here

Reply:
That's what this section is. Or do you mean the term "parameterization"?

Reply:
p.s. we don't have that term anywhere in docs anymore BTW.

Reply:
Yeah I guess you mean the term. Added to the note for now.